docs/reference/module-api/module-overview.md

```diff
@@ -758,7 +765,7 @@ A route in the `apiRoutes`, `restApiRoutes`, `renderRoutes`, and `routes` sectio
 | Property | What is it? |
 | -------- | ----------- |
 | `route` | A route function that accepts a request as its argument |
-| `before` | Identify a module's middleware or routes. Used if the route function should be registered before particular middleware or other routes. Format as `middleware:nameOfModule` for middleware or `nameOfModule` for routes. |
+| `before` | Identify another module's middleware or routes. Used if the route function should be registered before particular middleware or other routes. Format as `middleware:nameOfModule` for middleware or `nameOfModule` for routes. |
```

Comment:
How about "registered before a particular module's middleware or routes." Routes are not "other" middleware.